packages/labs/ssr/src/lib/render-value.ts

```diff
@@ -302,16 +307,22 @@ const getTemplateOpcodes = (
   );
 
   const hydratable = isHydratable(result);
+  const isServerTemplate = !hydratable;
```

Comment:
Nit: the rest of the code uses `hydratable` directly as basically meaning server template. e.g. https://github.com/lit/lit/blob/9c02b3876dc927c6a82b4420411256ecbb47c08c/packages/labs/ssr/src/lib/render-value.ts#L460-L470

I don't know why the naming was chosen as such then, but I don't see a need to introduce another boolean, even though I much prefer the name `isServerTemplate`.

Reply:
Done!